src/bidiMapper/domains/log/logManager.ts

```diff
@@ -162,8 +158,8 @@ export class LogManager {
     return 'info';
   }
 
-  // convert CDP StackTrace object to Bidi StackTrace object
-  static #getBidiStackTrace(
+  /** Converts CDP StackTrace object to Bidi StackTrace object. */
+  #getBidiStackTrace(
```

Comment:
I think this has to be a module-level function? i.e., if a function does not use any state on the instance it should be a standalone function

Reply:
Done